mono/tests/Makefile.am

```diff
@@ -1,19 +1,38 @@
 SUBDIRS = assemblyresolve gc-descriptors . testing_gac
 
-if FULL_AOT_TESTS
-FEATUREFUL_RUNTIME_TEST =  
-else
-FEATUREFUL_RUNTIME_TEST = test-appdomain-unload
+check-local:
+	$(MAKE) test-jit
+	$(MAKE) test-generic-sharing
+	$(MAKE) test-type-load
+	$(MAKE) test-multi-netmodule
+	$(MAKE) test-cattr-type-load
+	$(MAKE) test-reflection-load-with-context
+	$(MAKE) test-platform
+	$(MAKE) test-console-output
+	$(MAKE) test-env-options
+	$(MAKE) test-unhandled-exception-2
+if !FULL_AOT_TESTS
+	$(MAKE) test-appdomain-unload
 endif
+	$(MAKE) test-process-stress
+	$(MAKE) test-pedump
+	$(MAKE) rm-empty-logs
+	$(MAKE) runtest-gac-loading
 
```

Comment:
What is the purpose of this change ? Why can't we use normal make dependencies ?